scripts/randdisable

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env perl
+# Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+#
+# This script makes a "random" build using configure and verifies that it
+# builds curl correctly. It randomly adds a number of the available
+# --disable-* flags to configure. When it detects a problem the script stops,
+# otherwise it continues trying more combinations.
+#
+# 1. Figure out all existing configure --disable-* options
+# 2. Generate random command line using supported options + random TLS
+# 3. Run configure (exit if problem)
+# 4. run "make -sj10" to build (exit if problem)
+# 5. run curl -V (exit if problem)
+# 6. GOTO 2
+#
+# Tips:
+#
+# - edit the @tls array to include all TLS backends you can build with
+# - do a checkout in a ram-based filesystem
+#
+use List::Util qw/shuffle/;
+
+sub getoptions {
+    my @all = `./configure --help`;
+    for my $o (@all) {
+        chomp $o;
+        if($o =~ /(--disable-[^ ]*)/) {
+            if($1 !~ /FEATURE/) {
+                push @disable, $1;
+            }
+        }
+    }
+}
+
+getoptions();
+
+# options to select a TLS
+my @tls = ("--with-openssl",
+           "--with-wolfssl=/home/daniel/build-wolfssl",
```

Comment:
```suggestion
           "--with-wolfssl=" . $ENV{'HOME'} . "/build-wolfssl",
```
in one of the Linux CI jobs at least. Probably needs tweaking more.

Reply:
well, these are options that need manual tweaking to work properly for a given build so I wasn't sure how much of it I should just leave and how to show them off...

Reply:
I don't plan to add this in any CI